2018-12-09 18:25:29,589 ERROR: Exception on /login [GET] [in /mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/routes.py", line 83, in login
    return render_template('login.html', title='Sign In', form=form)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/templates/base.html", line 60, in template
    {% block content %}{% endblock %}
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/templates/login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/templates/base.html", line 60, in template
    {% block content %}{% endblock %}
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 543, in _generate
    optimized=self.optimized)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/compiler.py", line 82, in generate
    generator.visit(node)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/compiler.py", line 718, in visit_Template
    self.fail('block %r defined twice' % block.name, block.lineno)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/compiler.py", line 315, in fail
    raise TemplateAssertionError(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateAssertionError: block 'content' defined twice
2018-12-09 18:29:39,977 INFO: Microblog startup [in /mnt/d/diploma/flask_mega_tutorial/microblog/app/__init__.py:46]
2018-12-09 18:29:44,379 ERROR: Exception on /login [GET] [in /mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/routes.py", line 83, in login
    return render_template('login.html', title='Sign In', form=form)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/templates/base.html", line 60, in template
    {% block content %}{% endblock %}
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/templates/login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/app/templates/base.html", line 60, in template
    {% block content %}{% endblock %}
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 543, in _generate
    optimized=self.optimized)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/compiler.py", line 82, in generate
    generator.visit(node)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
    return f(node, *args, **kwargs)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/compiler.py", line 718, in visit_Template
    self.fail('block %r defined twice' % block.name, block.lineno)
  File "/mnt/d/diploma/flask_mega_tutorial/microblog/venv/lib/python3.6/site-packages/jinja2/compiler.py", line 315, in fail
    raise TemplateAssertionError(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateAssertionError: block 'content' defined twice
